docs/Hardware-Agnostic-Training.md

```diff
@@ -0,0 +1,67 @@
+# Switching Hardware Mid Training Run
+
+One of the standout features of Levanter is its ability to seamlessly operate on both GPU and TPU platforms. This guide will walk you through the steps and best practices for switching hardware configurations mid-training, ensuring a smooth and efficient training process for your models.
```

Comment:
maybe add a bit here or elsewhere about how changing the number or kind of accelerators midrun (while preserving optimizer state) isn't something most frameworks support?